documentation/modules/upgrading/proc-downgrade-brokers-older-kafka.adoc

```diff
@@ -8,30 +8,11 @@
 
 This procedure describes how you can downgrade a Strimzi Kafka cluster to a lower (previous) version of Kafka, such as downgrading from {KafkaVersionHigher} to {KafkaVersionLower}.
 
-[IMPORTANT]
-====
-Downgrading is _not possible_ if the new version has ever used a `log.message.format.version` that is not supported by the previous version, including when the default value for `log.message.format.version` is used. For example, this resource can be downgraded to Kafka version {KafkaVersionLower} because the `log.message.format.version` has not been changed:
-
-[source,yaml,subs=attributes+]
-----
-apiVersion: v1alpha1
-kind: Kafka
-spec:
-  # ...
-  kafka:
-    version: {KafkaVersionHigher}
-    config:
-      log.message.format.version: "{LogMsgVersLower}"
-      # ...
-----
-
-The downgrade would not be possible if the `log.message.format.version` was set at `"{LogMsgVersHigher}"` or a value was absent (so that the parameter took the default value for a {KafkaVersionHigher} broker of {LogMsgVersHigher}).
-====
-
 .Prerequisites
 
 For the `Kafka` resource to be downgraded, check:
 
+* [IMPORTANT] xref:com-target-downgrade-version-{context}[Compatibility of Kafka versions].
 * The Cluster Operator, which supports both versions of Kafka, is up and running.
 * The `Kafka.spec.kafka.config` does not contain options that are not supported in the version of Kafka you are downgrading to.
```

Comment:
_Suggested change:_

Use the same phrase in lines 17 and 18:

Kafka version being downgraded to

Reply:
:+1:  I changed both to: _by the Kafka version being downgraded to._